rmgpy/data/solvation.py

```diff
@@ -60,7 +60,31 @@ def save_entry(f, entry):
     f.write('    index = {0:d},\n'.format(entry.index))
     f.write('    label = "{0}",\n'.format(entry.label))
 
-    if isinstance(entry.item, Species):
+    if isinstance(entry.item, list):
+        if len(entry.item) == 1:
+            item = entry.item[0]
+            if isinstance(item, Species):
+                if Molecule(smiles=item.molecule[0].to_smiles()).is_isomorphic(item.molecule[0]):
+                    # The SMILES representation accurately describes the molecule, so we can save it that way.
+                    f.write('    molecule = "{0}",\n'.format(item.molecule[0].to_smiles()))
+                else:
+                    f.write('    molecule = \n')
+                    f.write('"""\n')
+                    f.write(item.to_adjacency_list(remove_h=False))
+                    f.write('""",\n')
+            else:
+                raise DatabaseError("Not sure how to save {0!r}".format(entry.item))
+        else:
+            spc_list = []
+            for item in entry.item:
+                if isinstance(item, Species):
+                    # If the entry.item is a list of more than one species, they must be saved as SMILES
+                    if Molecule(smiles=item.molecule[0].to_smiles()).is_isomorphic(item.molecule[0]):
+                        spc_list.append(item.molecule[0].to_smiles())
+                    else:
+                        raise DatabaseError("Not sure how to save {0!r}".format(entry.item))
```

Comment:
You can fall back to adjlist if it's one species, but you can't when the solvent involves multiple species?

Reply:
Yeah, for now, I made it possible to only save a list of solvents (multiple solvents) as a list of SMILES. We only have two types of multiple solvents in RMG-dabase, methanol + water (["CO", "O"]) and acetonitrile + water (["CC#N", "O"]), which are all stored as a list of SMILES so I thought it wold be fine.

I will also add an option of saving as a list of SMILES or adjlist for multiple solvents as well for the potential future use.

Reply:
I change the code so that it now allows an option of falling back to adjlist for the solvent involving multiple species. (fixup commit 64cf75)
I didn't create a unittest for this because it requires writing a file, but I manually checked to make sure this works properly.